app/scripts/controllers/reviewRegistration.js

```diff
@@ -68,98 +68,55 @@ angular.module('confRegistrationWebApp')
       return _.find($scope.blocks, {id: blockId});
     };
 
-    $scope.confirmRegistration = function () {
-      $scope.submittingRegistration = true;
-
-      /*if the totalPaid (previously) AND the amount of this payment are less than the minimum required deposit, then
-        show and error message. the first payment must be at least the minimum deposit amount.  subsequent payments
-        can be less than the amount.  this is confirmed by making sure the total previously paid is above the min deposit amount.
-        */
-      if ($scope.currentRegistration.pastPayments.length === 0 && Number($scope.currentPayment.amount) < $scope.currentRegistration.calculatedMinimumDeposit) {
-        $scope.currentPayment.errors.push('You are required to pay at least the minimum deposit of ' + $filter('currency')(registration.calculatedMinimumDeposit, '$') + ' to register for this event.');
+    $scope.getConfirmButtonName = function () {
+      if (currentRegistration.completed || !$scope.spouseRegistration) {
+        return gettextCatalog.getString('Confirm');
```

Comment:
If the existing code uses gettextCatalog, maybe that's an old way of doing it. Idk. Doug's comment makes is sound like we shouldn't use that. Brian Zoetewey who I think has done most of the translation in other apps has used `gettext('string')` a lot. He'd be a good person to ask if you're still unsure. I've never actually run the translations, I've just added the annotations.